sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/nlp/nlp_search_space.py

```diff
@@ -39,11 +38,6 @@ def __init__(
                 for item in learning_rate_scheduler.values
             ]
 
-        if isinstance(model_name, NlpModels):
```

Comment:
qq: where does model_name get added now?

Reply:
it is not shown in the diff, but if you view file you can see `self.model_name = model_name` on line 44

Reply:
I restored this file for backcompat as well